src/components/Dashboard/TalkDashboard.vue

```diff
@@ -0,0 +1,435 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<script lang="ts" setup>
+import { ref, onMounted, computed, onBeforeUnmount, watch, nextTick } from 'vue'
+import { useRouter } from 'vue-router/composables'
+
+import IconAlarm from 'vue-material-design-icons/Alarm.vue'
+import IconArrowLeft from 'vue-material-design-icons/ArrowLeft.vue'
+import IconArrowRight from 'vue-material-design-icons/ArrowRight.vue'
+import IconAt from 'vue-material-design-icons/At.vue'
+import IconCalendarBlank from 'vue-material-design-icons/CalendarBlank.vue'
+import IconMicrophone from 'vue-material-design-icons/Microphone.vue'
+import IconVideo from 'vue-material-design-icons/Video.vue'
+
+import { showError } from '@nextcloud/dialogs'
+import { emit } from '@nextcloud/event-bus'
+import { t } from '@nextcloud/l10n'
+
+import NcButton from '@nextcloud/vue/components/NcButton'
+import NcEmptyContent from '@nextcloud/vue/components/NcEmptyContent'
+
+import EventCard from './EventCard.vue'
+import ConversationsListVirtual from '../LeftSidebar/ConversationsList/ConversationsListVirtual.vue'
+import SearchMessageItem from '../RightSidebar/SearchMessages/SearchMessageItem.vue'
+import LoadingPlaceholder from '../UIShared/LoadingPlaceholder.vue'
+
+import { useStore } from '../../composables/useStore.js'
+import { CONVERSATION } from '../../constants.ts'
+import { hasTalkFeature } from '../../services/CapabilitiesManager.ts'
+import { useTalkDashboardStore } from '../../stores/talkdashboard.ts'
+import type { Conversation, DashboardEventRoom, UpcomingReminder } from '../../types/index.ts'
+import { filterConversation } from '../../utils/conversation.ts'
+
+const supportsUpcomingReminders = hasTalkFeature('local', 'upcoming-reminders')
+
+const FIVE_MINUTES = 5 * 60 * 1000 // 5 minutes
+const store = useStore()
+const router = useRouter()
+const talkDashboardStore = useTalkDashboardStore()
+const forwardScrollable = ref(false)
+const backwardScrollable = ref(false)
+const eventCardsWrapper = ref<HTMLInputElement | null>(null)
+const eventRooms = ref<DashboardEventRoom[]>(talkDashboardStore.getEventRooms)
+const upcomingReminders = ref<UpcomingReminder[]>(talkDashboardStore.getUpcomingReminders)
+const eventsInitialised = ref(eventRooms.value.length > 0)
+const remindersInitialised = ref(upcomingReminders.value.length > 0)
```

Comment:
What's the point in the store then, if you request information on mount?
```suggestion
const eventRooms = computed(() => talkDashboardStore.eventRooms)
const upcomingReminders = computed(() => talkDashboardStore.upcomingReminders)
const eventsInitialised = computed(() => talkDashboardStore.eventRoomsInitialised)
const remindersInitialised = computed(() => talkDashboardStore.upcomingRemindersInitialised)
```